x-pack/plugins/cases/common/api/saved_object.ts

```diff
@@ -19,52 +19,3 @@ export const NumberFromString = new rt.Type<number, string, unknown>(
     }),
   String
 );
-
-const ReferenceRt = rt.type({ id: rt.string, type: rt.string });
-
-export const SavedObjectFindOptionsRt = rt.partial({
```

Comment:
I was surprised this was not used anywhere else.

I created `FindCommentsQueryParamsRt` which is more specific.

Reply:
Yes, I removed most of the usage in https://github.com/elastic/kibana/pull/155898. Glad to see this removed 🙂.